src/emulator/functionsEmulator.ts

```diff
@@ -1631,7 +1788,7 @@ export class FunctionsEmulator implements EmulatorInstance {
         return;
       }
     }
-    let debugBundle;
+    let debugBundle: any;
```

Comment:
Would it be possible to remove this `any`? I think it could be typed as this?

https://github.com/firebase/firebase-tools/blob/bfb9c32188c272409b59d93e0bdd162a52f9c2c7/src/emulator/functionsEmulatorShared.ts#L88

Reply:
`let debugBundle: FunctionsRuntimeBundle[debug]` or something like that